### PR TITLE
Blog Post Page

### LINKED ISSUE #1: Blog
Once we have the content processing pipeline in place. This should be just a matter of converting the existing content and displaying them in a similar manner as we have today.

Depends on: #16

### PR DESCRIPTION
**Pull request by KaneFreeman**
## Description

- Add blog post page.
- Clean up CSS in blog index page to match blog post page.
- Remove the use of `i18n.locale`. Simply default to `en` as the language for now.

Resolved #1 

### Code

This PR touches:

- [ ] Content
- [ ] Content Pipeline
- [x] Frontend
- [ ] Infrastructure

### Tests

- [x] Tests are included?

### Screenshots

image
blog-section

### CODE REVIEW

File: src/main.css [hunk 1]
```diff
@@ -1,12 +1,14 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat');
```
Comment (JamesMilnerUK) [inline src/main.css]: More of a query, do we want to use Google Fonts in production or?

File: src/pages/Blog.m.css [hunk 2]
```diff
@@ -1,3 +1,3 @@
 .root {
-	padding: var(--content-top-padding);
+	width: 1000px!important;
```
Comment (JamesMilnerUK) [inline src/pages/Blog.m.css]: Is there a specific reason we're using 1000px and does it have to be `!important`?

File: src/widgets/code/CodeSandbox.spec.tsx [hunk 3]
```diff
@@ -27,22 +27,14 @@ const mockMetaMixin = <T extends Constructor<WidgetBase<any>>>(Base: T, mockStub
...
-		));
-	});
-
-	it('renders with build-time-render', () => {
 		const mockMeta = jest.fn().mockImplementation((input: any) => {
 			if (Intersection) {
-				return {
-					get: mockIntersectionGet
-				};
+					return {
```
Comment (JamesMilnerUK) [inline src/widgets/code/CodeSandbox.spec.tsx]: This indent looks pretty intense. Is it right?

File: src/widgets/page/Page.m.css [hunk 4]
```diff
@@ -12,14 +12,13 @@
...
-	min-width: 900px;
-	width: 50%;
+	width: 1000px;
 	margin: 0 auto 80px;
 }
 
 .footer {
 	color: #999 !important;
-	font-size: 16px;
+	font-size: 14	px;
```
Comment (JamesMilnerUK) [inline src/widgets/page/Page.m.css]: whitespace

File: src/widgets/page/Page.m.css [hunk 5]
```diff
@@ -12,14 +12,13 @@
...
 .content {
 	flex-grow: 1;
-	min-width: 900px;
-	width: 50%;
+	width: 1000px;
 	margin: 0 auto 80px;
 }
 
 .footer {
 	color: #999 !important;
```
Comment (JamesMilnerUK) [inline src/widgets/page/Page.m.css]: I know this isn't your code but is there a reason we are using a hex variable and `!important` ?

File: src/main.css (hunk 1 above)
Comment (KaneFreeman) [reply src/main.css]: I can pull it down locally. That was the font that disappeared though 😄

File: src/pages/Blog.m.css (hunk 2 above)
Comment (KaneFreeman) [reply src/pages/Blog.m.css]: 1000? no, it just looks good. Important? Yes, as with JS off the CSS loading order gets messed up resulting in the Landing widget's default with taking back over.

File: src/widgets/code/CodeSandbox.spec.tsx (hunk 3 above)
Comment (KaneFreeman) [reply src/widgets/code/CodeSandbox.spec.tsx]: Nope. Hadn't run prettier yet.

**Pull request merged**